tsconfig.json

```diff
@@ -14,7 +14,6 @@
   },
   "include": [
     "src/**/*",
-    "test/**/*",
```

Comment:
I don't remember why I had to remove this :/ I'll try adding it back and see if everything works